engine/consensus/ingestion/core.go

```diff
@@ -57,6 +57,8 @@ func NewCore(
 // All other errors are unexpected and potential symptoms of internal state corruption.
 func (e *Core) OnGuarantee(originID flow.Identifier, guarantee *flow.CollectionGuarantee) error {
 
+	e.log.Info().Msgf("on guarantee called for guarantee %v from %v", guarantee.ID(), originID)
```

Comment:
```suggestion
	e.log.Debug().Msgf("on guarantee called for guarantee %v from %v", guarantee.ID(), originID)
```
Not sure if we want info-level for this common of an event